src/theory/strings/theory_strings.cpp

```diff
@@ -189,18 +190,20 @@ bool TheoryStrings::propagate(TNode literal) {
   return ok;
 }
 
-
-Node TheoryStrings::explain( TNode literal ){
+TrustNode TheoryStrings::explain(TNode literal)
+{
   Debug("strings-explain") << "explain called on " << literal << std::endl;
   std::vector< TNode > assumptions;
   d_im->explain(literal, assumptions);
+  Node ret;
```

Comment:
You could remove the block below and do:
```suggestion
  Node ret;
  ret = assumptions.empty()
            ? d_true
            : assumptions.size() == 1
                  ? assumptions[0]
                  : NodeManager::currentNM()->mkNode(kind::AND, assumptions);
```
If you don't want to then could you clang-format it, as you are already changing most of the method?

Reply:
This code is deleted in proof-new, let's not worry about this.